src/lesson/lesson01.js

```diff
@@ -469,11 +469,16 @@ function Lesson01() {
           pode apertar o {Icons.HelpIcon}. Minha principal lição é: não entre em pânico!
         </p>
       </div>,
+      [],
```

Comment:
Was it intentional that the first step of this lesson doesn't have the reference?